buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginBuildPlugin.groovy

```diff
@@ -51,21 +52,36 @@ public class PluginBuildPlugin extends BuildPlugin {
             String name = project.pluginProperties.extension.name
             project.archivesBaseName = name
 
-            // set teh project description so it will be picked up by publishing
+            // set the project description so it will be picked up by publishing
             project.description = project.pluginProperties.extension.description
 
             configurePublishing(project)
 
-            project.integTestCluster.dependsOn(project.bundlePlugin)
+            if (project.plugins.hasPlugin(TestClustersPlugin.class) == false) {
+                project.integTestCluster.dependsOn(project.bundlePlugin)
```

Comment:
FWIW, in Groovy plugins where we don't use static compilation, it's nice to be extra explicit with dynamic stuff. For example, in this case using `project.tasks.bundlePlugin` makes it a bit more obvious to the ready what `bundlePlugin` is.